README.md

```diff
@@ -40,7 +40,7 @@ release.**
 
 ## Building the Jobs library
 
-A compiler that supports **C90 or later** such as _gcc_ is required to build the
+A compiler that supports **C99 or later** such as _gcc_ is required to build the
```

Comment:
Why is C99 needed? Can this code not support C90?

Reply:
use of stdbool.h which was introduced in c99

Reply:
Wait does this PR add `stdbool.h` or is this something that was already in the repo and we didn't account for it in this README?

Reply:
This PR adds stdbool.h